wrappers/csharp/Intel.RealSense/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required( VERSION 3.8.0 )
+cmake_minimum_required( VERSION 3.1.0 )
```

Comment:
should be 3.8.0

Reply:
👍

Reply:
Actually, the pInvoke is built without explicit usage of  project(LANGUAGES CSharp), so v3.1 would probably suffice.
I'll modify it in any event.